.changeset/tender-ghosts-obey.md

```diff
@@ -0,0 +1,5 @@
+---
+"@replayio/sourcemap-upload": patch
```

Comment:
Even though the other package will already benefit automatically~ from this change, I'd probably call it out explicitly here:
```suggestion
"@replayio/sourcemap-upload": patch
"@replayio/sourcemap-upload-webpack-plugin": patch
```